.github/workflows/spellcheck.yml

```diff
@@ -14,50 +15,26 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  roxygen:
-    name: Spellcheck 🔠
-    runs-on: ubuntu-20.04
-    env:
-      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+  spelling:
+    name: Run Spellcheck 🔠
+    runs-on: ubuntu-latest
     if: >
       !contains(github.event.commits[0].message, '[skip spellcheck]')
         && github.event.pull_request.draft == false
     steps:
-      - name: Checkout repo 🛎
-        uses: actions/checkout@v3
-        with:
-          persist-credentials: false
-          fetch-depth: 0
+      - name: Checkout repository 🛎
+        uses: actions/checkout@v4
 
       - name: Setup R 📊
         uses: r-lib/actions/setup-r@v2
-        with:
-          r-version: 4.1.3
 
-      - uses: actions/cache@v2
-        if: startsWith(runner.os, 'Linux')
+      - name: Install R package dependencies 📦
+        uses: r-lib/actions/setup-r-dependencies@v2
         with:
-          path: ~/.local/share/renv
-          key: ${{ runner.os }}-renv-${{ hashFiles('**/renv.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-renv-
-      - name: Query dependencies
-        run: |
-          install.packages('remotes')
-          saveRDS(remotes::dev_package_deps(dependencies = TRUE), ".github/depends.Rds", version = 2)
-          writeLines(sprintf("R-%i.%i", getRversion()$major, getRversion()$minor), ".github/R-version")
-        shell: Rscript {0}
-
-      - name: Install system dependencies
-        if: runner.os == 'Linux'
-        run: |
-          while read -r cmd
-          do
-            eval sudo $cmd
-          done < <(Rscript -e 'writeLines(remotes::system_requirements("ubuntu", "20.04"))')
+          extra-packages: any::spelling
 
       - name: Run Spellcheck 👟
-        uses: insightsengineering/r-spellcheck-action@v3
+        uses: insightsengineering/r-spellcheck-action@v2
```

Comment:
Is this downgrade from v3 intentional?

Reply:
No it wasn't. Copy paste failure from the [example on the repo](https://github.com/insightsengineering/r-spellcheck-action?tab=readme-ov-file#usage)

corrected with 4933aa5

Reply:
However, it seems like `v3` has a bug. I've submitted a bug there: https://github.com/insightsengineering/r-spellcheck-action/issues/9

Reply:
Solved upstream